manifests/repo/debian.pp

```diff
@@ -24,16 +24,16 @@
 #   Enable special sury handling
 #
 class php::repo::debian(
-  $location     = 'http://packages.dotdeb.org',
-  $release      = 'wheezy-php56',
-  $repos        = 'all',
-  $include_src  = false,
-  $key          = {
+  String           $location     = 'http://packages.dotdeb.org',
+  Optional[String] $release      = undef,
+  String           $repos        = 'all',
+  Boolean          $include_src  = false,
+  Hash             $key          = {
     'id'     => '6572BBEF1B5FF28B28B706837E3F070089DF5277',
     'source' => 'http://www.dotdeb.org/dotdeb.gpg',
   },
-  $dotdeb       = true,
-  $sury         = true,
+  Boolean          $dotdeb       = false,
+  Boolean          $sury         = false,
```

Comment:
Bc break. both bool flags should stay on true or if the `sury` parameter has not be released, we can change sury to false as default